source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,130 @@
+===========
+Aggregation
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **aggregation operations** in
+the MongoDB Node.js driver.
+
+Aggregation operations are expressions you can use to produce summarized
+and computed results in MongoDB. MongoDB's aggregation framework
+allows you to create a pipeline that consists of one or more stages,
+each of which performs a specific operation on your data.
+
+You can think of the aggregation framework as similar to an automobile factory.
+Automobile manufacturing requires the use of assembly stations organized
+into assembly lines. Each station has specialized tools, such as
+drills, welders, or other machinery. The factory transforms and
```

Comment:
```suggestion
into assembly lines. Each station has specialized tools to do a specific job, such as
drills and welders. The factory transforms and
```

I think "other machinery" should be removed. I think "machinery" is a different category than "tool", and I think "other machinery" is vague.

Reply:
ok